gremlin-server/src/test/java/org/apache/tinkerpop/gremlin/server/GremlinDriverIntegrateTest.java

```diff
@@ -938,7 +938,8 @@ public void shouldWorkWithGraphSONExtendedV3Serialization() throws Exception {
 
     @Test
     public void shouldFailClientSideWithTooLargeAResponse() {
-        final Cluster cluster = TestClientFactory.build().maxContentLength(1).create();
+        final int maxContentLength = 128;
```

Comment:
I assume you needed to change this because the validation message is now being sent and it's greater than "1" in size?

Reply:
Yes, it is.